lib/process/spawn.fz

```diff
@@ -100,6 +118,11 @@ private:public Process(id i64, std_in option i64, std_out, std_err i64) ref is
       .with f
 
 
+  public with_in(T type, LM type : mutate, f ()->T) outcome T =>
```

Comment:
What is the value of passing a custom mutate here?

Reply:
You could create a custom local mutate like it is done in other places in the stdlib / tests.
```
lm : mutate is
(io.buffered.writer lm (std_in_write_provider) pipe_buffer_size)
      .with f
```

Reply:
Ah okay. Then it would probably be good to have one version using a predefined type like with `open_unique_type` but this does not need to be part of this PR.